.github/workflows/node.js.yml

```diff
@@ -43,7 +43,7 @@ jobs:
         with:
           node-version: ${{ matrix.node }}
       - name: Install dependencies
-        run: npm ci
+        run: npm config set legacy-peer-deps true &&  npm ci
```

Comment:
Why u add this?

Reply:
see description

Reply:
I added it to solve a problem that appeared when I installed @material-ui/core  package.